Source/ASDisplayNode+Layout.mm

```diff
@@ -579,6 +579,7 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
 
       BOOL automaticallyManagesSubnodesDisabled = (self.automaticallyManagesSubnodes == NO);
       self.automaticallyManagesSubnodes = YES; // Temporary flag for 1.9.x
+      newLayoutVersion = _layoutVersion;
```

Comment:
Can you set this immediately after the lock is taken so it is more clear why it is separate from its definition?